neqo-transport/src/connection/tests/datagram.rs

```diff
@@ -224,6 +222,48 @@ fn datagram_acked() {
     ));
 }
 
+#[test]
+fn datagram_after_stream_data() {
+    let (mut client, mut server) = connect_datagram();
+
+    // Wriate a datagram first.
+    let _dgram_sent = client.stats().frame_tx.datagram;
```

Comment:
Do you plan to use this variable?  You could check that it increases as an alternative way to verify things (then you might not need to check receipt at the server end at all (as noted below).

Reply:
I'll use it for testing. Thanks.